docs/item-service.md

```diff
@@ -0,0 +1,35 @@
+# Item service
+
+Manages the description and auction status (created, auction, completed, cancelled) of an item.
+
+## Queries handled
+
+* **getItem** - Gets an item by an ID.
+* **getItemsForUser** - Gets a list of items in a provided status that are owned by a given user.
+
+## Events emitted
+
+* **AuctionStarted** (public) - When the auction is started, in response to **startAuction**.
+* **ItemUpdated** (private) - When user editable fields on an item are updated in response to **createItem**.
```

Comment:
Hmmm.... I had use [ItemEvent.scala#L13](https://github.com/lagom/online-auction-scala/blob/5aa325ae3a5a0d603990a945d99044167eec95e6/item-api/src/main/scala/com/example/auction/item/api/ItemEvent.scala#L13) as my reference and the event is listed in the current README.md